sdk/python/kfp/dsl/pipeline_task.py

```diff
@@ -125,12 +126,14 @@ def __init__(
 
         self.component_spec = component_spec
 
+        self.caching_options = not os.getenv('KFP_DISABLE_CACHING', 'false').lower() == 'true'
```

Comment:
does this need to be set to `self.caching_options`? I was thinking you could just inline it like you did below

Reply:
Done, updated to have it inline.